gpt_index/indices/query/knowledge_graph/query.py

```diff
@@ -93,6 +154,9 @@ def _get_nodes_for_response(
         # filter sorted nodes
         sorted_nodes = [node for node in sorted_nodes if self._should_use_node(node)]
         for chunk_idx, node in zip(sorted_chunk_indices, sorted_nodes):
+            # nodes are found with keyword mapping, give high conf to avoid cutoff
+            if similarity_tracker is not None:
+                similarity_tracker.add(node, 1000.0)
```

Comment:
this is a bit hacky (the num 1000 is a bit arbitrary), but it's ok, i'll take a stab at this later